wdio.conf.js

```diff
@@ -169,7 +177,7 @@ exports.config = {
 
   mochaOpts: {
     ui: 'bdd',
-    timeout: 99999999,
+    timeout: 60000,
```

Comment:
will this be enough? I'm just curious.

Reply:
Typically yes, but we can increase this to 100000